apps/golden-sample-app/src/app/app-routing.module.ts

```diff
@@ -37,17 +37,17 @@ const routes: Routes = [
       entitlements: triplets.canViewPositivePay,
     },
   },
-  {
-    path: 'ach-positive-pay',
-    loadChildren: () =>
-      import('./ach-positive-pay/ach-positive-pay-journey-bundle.module').then(
-        (m) => m.AchPositivePayJourneyBundleModule
-      ),
-    canActivate: [AuthGuard, EntitlementsGuard],
-    data: {
-      entitlements: triplets.canViewAchRule,
-    },
-  },
+  // {
```

Comment:
Do we need these lines?

Reply:
Yes, we will keep it until the journey team will migrate to v7